package.json

```diff
@@ -27,4 +27,4 @@
     "acorn": "~8.2.4",
     "optimist": "~0.6.1"
   }
-}
+}
```

Comment:
nit: newline maybe (not used to github's UI for reviews)

Reply:
oh sure, I blame my editor for that one :)